### DISCUSSION TITLE
Using build function from mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2019 throws an error

### DISCUSSION THREAD

**Discussion by Rishitha-09** [microsoft/dotnet-framework-docker#902] [General]
Hi, I am trying to build a dockerfile using windows image,but when I try to use the build function it throws an error.

The error is:
![Screenshot (11)](https://user-images.githubusercontent.com/84839948/157476771-61e02d1a-35ea-41ca-895e-5da5de946db3.png)
And I also tried it by switching to windows container. When I was trying to install some packages and tools through docker file I am unable to install them using that windows image but its working through commandline.

My Dockerfile is:

FROM mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2019
RUN mkdir C:\Users\Devops
ENV AWS_ACCESS_KEY_ID=xxxxxxxxxxxx
ENV AWS_SECRET_ACCESS_KEY=xxxxxxxxxxxxx
ENV AWS_DEFAULT_REGION=us-east-1
RUN choco update && choco install -y awscli
RUN aws s3 cp s3://tecmfa123/nuget.exe C:\Users\Devops
RUN choco install nuget -y
RUN nuget install Costura.Fody
RUN aws s3 cp s3://tecmfa123/wix311.exe C:\Users\Devops
RUN aws s3 cp s3://tecmfa123/Votive2019.vsix C:\Users\Devops
RUN aws s3 cp s3://tecmfa123/VSIXInstaller.exe C:\Users\Devops
RUN wix311.exe /install /quiet /norestart
RUN VSIXInstaller.exe Votive2017.vsix /quiet

Can Someone help me?

Thanks

**Comment by mthalman** [comment]
This Dockerfile isn't valid. You're running `choco` and `aws` commands which don't exist from the `mcr.microsoft.com/dotnet/framework/sdk:4.8-windowsservercore-ltsc2019` base image. In addition, using `&&` in `RUN choco update && choco install -y awscli` line 6 isn't valid since the shell is PowerShell, not cmd. Please provide an updated Dockerfile that actually reproduces the issue.
